alerting-config-service-impl/src/main/java/org/hypertrace/alerting/config/service/AlertingConfigServiceConstants.java

```diff
@@ -0,0 +1,6 @@
+package org.hypertrace.alerting.config.service;
+
+public interface AlertingConfigServiceConstants {
```

Comment:
Do these need to be published as public constants? usually they're implementation details only relevant to one spot (which looks to be the case with your store here).

Reply:
No, moved them to EventStore